.github/quarkus-bot.yml

```diff
@@ -362,13 +362,9 @@ triage:
     - labels: [area/platform]
       directories:
         - independent-projects/tools/
-    - labels: [area/reactive]
+    - labels: [area/reactive-streams-operators]
```

Comment:
Do we still have reactive stream operators?

Reply:
I checked before keeping it and yes: https://github.com/quarkusio/quarkus/tree/main/extensions/reactive-streams-operators .